github/ci/prow-deploy/kustom/environments/kubevirtci-testing/kustomization.yaml

```diff
@@ -243,30 +256,36 @@ secretGenerator:
     files:
       - secrets/kubeconfig/config
     type: Opaque
-  - name: cookie
-    files:
-      - secrets/cookie-b64encoded/secret
-    type: Opaque
   - name: gcs
     files:
       - secrets/gcs/service-account.json
     type: Opaque
-  - name: github-oauth-config
```

Comment:
If you are removing these here, where are they set ?
This part should be the one responsible for creating the secret, so you know where to look
as single source of truth

Reply:
I see you're preparing some secrets in prepare.yaml now, I understand you would need a more dynamic way to create the secrets, but in this way you're making ansible essential. It's not possible to run this deployment manually now. if you intend to proceed , at least put a comment here clarifying where are those secrets defined.

Reply:
Sorry, i don't follow, why is ansible essential? If the files for kubeconfig, gcs and oauth-token are in place you can run kustomize, these changes only initialize some of the secrets for the testing environment. Also, the initialization in prepare.yml, as everything else under the molecule dir, are only meant for testing.

Reply:
OK, I think I understand now, You've not removed the secrets, you just changed their position inside the list, and this confused me, you're re-adding them hardcoded at the end.